frontends/mit-open/src/components/layout.tsx

```diff
@@ -15,7 +15,7 @@ type ColumnVariant =
   | "sidebar-2-wide-main"
 type GridItemProps = Omit<
   GridProps,
-  "item" | "container" | "xs" | "sm" | "md" | "lg" | "xl"
```

Comment:
Grid items can also be containers that themselves have items, so there was no reason to Omit `container`  from the props here.